sds_data_manager/constructs/instrument_lambdas.py

```diff
@@ -229,3 +235,26 @@ def __init__(
                     )
                 ],
             )
+
+        scheduled_rules = {
+            # Example:
+            # "glows": "cron(20 6 * * ? *)",
+            # "sp_maps": "cron(20 14 * * ? *)",
+        }
+
+        for name, schedule_expression in scheduled_rules.items():
+            aws_events.CfnRule(
```

Comment:
Rather than the level 1 construct `Cfn...` forms here, I think we'd want to use the level 2 constructs like: https://docs.aws.amazon.com/cdk/api/v2/python/aws_cdk.aws_events/Rule.html
Unless there is an explicit reason we need to dive down into the low-level Cfn formatting.